docs/sovereign/from-sovereign.md

```diff
@@ -0,0 +1,111 @@
+# Execution starting from a Sovereign Chain
+![From Sovereign](../../static/sovereign/from-sovereign.png)
+
+For the user — whether an External Owned Account (EOA) like a user wallet or another smart contract — procedure is simple. An address will be able to perform a multiTransfer deposit for various types of tokens:
+1. Fungible Tokens
+2. Non-Fungible Tokens
+3. Semi-Fungible Tokens
+4. Meta ESDT Tokens
+
+When making the deposit, the user specifies:
+1. A destination address on the Sovereign Chain
+2. The function to call
+3. Any required arguments
+4. *gasLimit* if a smart contract execution is needed after the cross-chain operation
+
+Each action that can be executed remotely through this contract is called an *Operation*. The endpoint responsible for executing those operations is called `execute_operations`.
+
+#### Execution from inside the Sovereign Chain flow
+1. User sends token to the ESDT-Safe smart contract on Sovereign.
+2. The validators generate a proof on the Sovereign Chain for a batch of transfers, which will be sent to the mainchain ESDT-Safe contract.
+3. Validators add this information to the sovereignChainBlockBody. Otherwise the block is not signed.
+4. Leader will push the created txData to the mainchain
```

Comment:
Outgoing operations will be created at the end of the round and will be signed by all the validators and sent to the bridge service by the leader.